packages/fluentui/react-builder/src/utils/codeToTree.ts

```diff
@@ -47,14 +48,26 @@ export const codeToTree: (code: string) => JSONTreeElement = code => {
       }
     }
 
+    // eslint-disable-next-line
+    const isElement = 'A' <= name[0] && name[0] <= 'Z';
+    const isComponent = isElement && (!props || (props && props['data-builder-id'] !== 'undefined'));
```

Comment:
It can happen the `data-builder-id="undefined"` in some element?

Reply:
Yes, it happens to Icons. The problem is that we are adding some margins and paddings to every element with `data-builder-id`. So for icons we want to keep it undefined. But for example if user created new component in code editor. We need to generate new uuid.

Reply:
Maybe use some random id instead of `undefined` to avoid react complain about  2 items with same key ?